### PR TITLE
UI-4470  jest - enzyme version change

### PR DESCRIPTION
**Pull request by sheela-svmx**
Upgrade enzyme and jest

Had to ignore test as they have been failing even before the upgrade

https://servicemax.atlassian.net/browse/UI-4745

Also I have a few questions

Should i do a version change  on package.json?

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@svmx/react-scripts",
+  "name": "@svmx/react-scripts-sheela",
```
Comment (svmx-pratima) [inline packages/react-scripts/package.json:2]: should it be "@svmx/react-scripts-sheela"?

Comment (sheela-svmx) [reply packages/react-scripts/package.json:2]: My bad it should not be. I use that to test my cra to run and make sure tests were ok

**Comment by mallik-svmx** [comment]
> Upgrade enzyme and jest
> 
> Had to ignore test as they have been failing even before the upgrade
> 
> https://servicemax.atlassian.net/browse/UI-4745
> 
> Also I have a few questions
> 
> Should i do a version change on package.json?

Nope. The build is configured to upgrade the package version automatically.

**Comment by svmx-pratima** [comment]
2 approvals-merging

**Pull request merged**